psalm-baseline.xml

```
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.11.1@e33492398bd4e5e2ab60e331d445979bd83feecd">
+    <file src="src/Controller/AuditController.php">
+        <UndefinedClass occurrences="1">
+            <code>Controller</code>
+        </UndefinedClass>
+    </file>
+    <file src="vendor/doctrine/orm/lib/Doctrine/ORM/Internal/SQLResultCasing.php">
```

this is weird

---

The class is internal, but IMO this should be added on code not here (thats how we did on other cases)

---

But why do we have to ignore an error for a vendor file ?
`vendor/doctrine/orm/lib/Doctrine/ORM/Internal/SQLResultCasing.php`

---

Oh, I don't know. I thought this was only because it was internal.

---

[`Doctrine\ORM\Internal\SQLResultCasing::getSQLResultCasing()`](https://github.com/doctrine/orm/blob/203cd6ecf256c43b4a9a7ca88a35ed7c87c585e1/lib/Doctrine/ORM/Internal/SQLResultCasing.php#L27) is using [`Doctrine\DBAL\Platforms\PostgreSQLPlatform`](https://github.com/doctrine/dbal/blob/75fe7631363e1957e087d03d09eb55b0bf8a493c/lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php#L37-L44). This class was deprecated in "doctrine/dbal" 2.11.0 and removed in version 3.0.

---

Yes, but psalm is not supposed to report errors in the vendor...

---

You need to ignoreFile part maybe https://github.com/sonata-project/SonataAdminBundle/blob/4.x/psalm.xml#L6-L9

---

Perfect. `vendor/` dir added to the ignore list :+1: